src/Sylius/Bundle/CoreBundle/Resources/config/app/config.yml

```diff
@@ -23,6 +23,9 @@ imports:
     - { resource: "@SyliusCoreBundle/Resources/config/app/state_machine.yml" }
     - { resource: "@SyliusCoreBundle/Resources/config/app/fixtures.yml" }
 
+parameters:
+    sylius_core.public_dir: "%kernel.project_dir%/web"
```

Comment:
This is of course wrong. But also, is it really necessary for this to be configurable?

Reply:
This is introduced only to make the changes backwards compatible - so that it won't fail if you upgrade from 1.2 to 1.3. It is overwritten in `config/_sylius.yaml` to `%kernel.project_dir%/public`.

Reply:
What do you mean by introducing more config files? `config/packages/_sylius.yaml` follows the Symfony 4 standard of having config files for each package. We can't remove the config files in bundles in `1.x` branch. As for configs in Sylius-Standard and PluginSkeleton and their synchronisation, this is a whole different issue to be handled and I believe doing it all at once (Symfony 4 directory structure, Symfony Flex, config files synchronisation) would result in not having anything that works (and if it does, it would probably result in some nasty BC breaks).